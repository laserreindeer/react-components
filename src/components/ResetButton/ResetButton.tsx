import { wrap } from "../../utils/wrap/wrap";

/**
 * ResetButton component using the wrap util.
 */
export const ResetButton = wrap("button")(props => (
	<button type="reset" {...props} />
));
