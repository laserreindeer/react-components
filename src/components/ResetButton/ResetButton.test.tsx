import { ResetButton } from "./ResetButton";
import { render } from "@testing-library/react";

it("matches snapshot", () => {
	const { asFragment } = render(<ResetButton>Test</ResetButton>);
	expect(asFragment()).toMatchSnapshot();
});
