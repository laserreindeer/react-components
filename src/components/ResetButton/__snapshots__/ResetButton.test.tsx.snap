// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <button
    type="reset"
  >
    Test
  </button>
</DocumentFragment>
`;
